guides/assets/esa-initial-flow.txt

```diff
@@ -0,0 +1,40 @@
+sequenceDiagram
+    participant User
+    participant App
+    participant Application Route (ApplicationRouteMixin)
+    participant Index Route (AuthenticatedRouteMixin)
+    participant Login Route (UnauthenticatedRouteMixin)
+    participant Session Service
+    participant Torii Provider
+    participant GitHub
+    participant Token Exchanger
+    User->>App: Use https://git-tool.com
+    App->>Application Route (ApplicationRouteMixin): beforeModel()
+    Application Route (ApplicationRouteMixin)->>Application Route (ApplicationRouteMixin): subscribeToSessionEvents()
+    App->>Index Route (AuthenticatedRouteMixin): beoreModel()
+    Index Route (AuthenticatedRouteMixin)->>Session Service: isAuthenticated?
+    Session Service-->>Index Route (AuthenticatedRouteMixin): false
+    Index Route (AuthenticatedRouteMixin)->>Login Route (UnauthenticatedRouteMixin): transitionTo(authenticationRoute)
+    Login Route (UnauthenticatedRouteMixin)->>Login Route (UnauthenticatedRouteMixin): beforeModel()
+    Login Route (UnauthenticatedRouteMixin)->>Session Service: isAuthenticated?
+    Session Service-->>Login Route (UnauthenticatedRouteMixin): false
+    Login Route (UnauthenticatedRouteMixin)-->>App: render(loginTemplate)
```

Comment:
I think I'd remove this as it might cause confusion. This is something that happens internally in Ember and that is also in no way overridden or hooked into by ESA so I don't think this needs to be shown at all.